service/create/service.go

```diff
@@ -94,24 +94,40 @@ func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
 		s.Config.Logger.Log("info", "Booted cert-operator")
 
-		s.Config.Logger.Log("info", "Test issuing a cert")
-
 		cert := CertificateSpec{
-			ClusterID:        "cert-test",
-			CommonName:       "api.cert-test.giantswarm.io",
-			IPSANs:           []string{"10.0.0.4", "10.0.0.5"},
-			AltNames:         []string{"api.k8s.cert-test.giantswarm.io"},
-			AllowBareDomains: false,
+			ClusterID:  "cert-test",
+			CommonName: "api.cert-test.g8s.eu-west-1.aws.test.private.giantswarm.io",
+			IPSANs:     []string{"10.0.0.4", "10.0.0.5"},
+			AltNames: []string{
+				"kubernetes",
+				"kubernetes.default",
+				"kubernetes.default.svc",
+				"kubernetes.default.svc.cluster.local",
+			},
+			AllowBareDomains: true,
 			TTL:              "720h",
 		}
 
-		issueResp, err := s.Issue(cert)
+		// Ensure a PKI backend exists for the cluster.
+		err := s.setupPKIBackend(cert)
 		if err == nil {
-			s.Config.Logger.Log("info", "Cert issued")
-			s.Config.Logger.Log("info", cert.CommonName)
-			s.Config.Logger.Log("info", issueResp.SerialNumber)
+			// Ensure a PKI policy exists for the cluster.
+			err := s.setupPKIPolicy(cert)
+			if err == nil {
```

Comment:
I noticed that this here is `err == nil`. It is usually `err != nil`. So this can easily be overlooked. I would like to agree on one way to handle error conditions to make our lives more easy. I think `err != nil` is more common. Lets create an issue for this and improve it separately.